CHANGELOG.rst

```diff
@@ -6,6 +6,39 @@ Unreleased `master`_
 
 :Date: YYYY-MM-DD
 
+Added
+~~~~~
+
+* Support for documenting C++ (contributions courtesy of `Critical Software`_)
```

Comment:
Not sure if you intend to merge this PR as it is for us to add things as we go along (not a bad idea by the way, and do so from now on for upcoming releases).

Anyway, if that's the case, I would only ask you to temper this sentence. Call it 'early support for' or 'cpp domain enablement' for now.

Reply:
I think I'll call it "early" or something for now, and we can adjust it later before the release, depending on what we have.

It would be great to update the changelog with the changes as they come instead of right before release, but nobody ever likes to write changelogs. :)